components/RichtextCollapsibleItem.js

```diff
@@ -7,16 +7,20 @@ import { styles } from './RichtextCollapsibleItem.styles';
 export const RichtextCollapsibleItem = ({ title, body }) => {
   const [collapsed, setCollapsed] = useState(true);
   const chevron = collapsed
```

Comment:
chevron and plus/minus? Can we call it sth. more general? Maybe collapseItem? I cannot think of sth. that makes looots of sense, but it's not a chevron anymore. :)